extensions/positron-python/python_files/posit/positron/positron_jedilsp.py

```diff
@@ -326,12 +326,22 @@ def start_tcp(self, host: str, port: int) -> None:
         self._stop_event = threading.Event()
```

Comment:
There's a log message up top about `logger.info("Starting TCP server on %s:%s", host, port)` that i think we should remove in favor of the new one below.